demo/executor/src/lib.rs

```diff
@@ -207,24 +259,39 @@ mod tests {
 			democracy: Some(Default::default()),
 			council: Some(Default::default()),
 			timestamp: Some(Default::default()),
-		}.build_storage().unwrap()
+		}.build_storage()
+			.unwrap()
 	}
 
-	fn construct_block(number: BlockNumber, parent_hash: Hash, state_root: Hash, extrinsics: Vec<BareExtrinsic>) -> (Vec<u8>, Hash) {
+	fn construct_block(
+		number: BlockNumber,
+		parent_hash: Hash,
+		state_root: Hash,
+		extrinsics: Vec<BareExtrinsic>,
+	) -> (Vec<u8>, Hash) {
 		use triehash::ordered_trie_root;
 
-		let extrinsics = extrinsics.into_iter().map(|extrinsic| {
-			let signature = MaybeUnsigned(Pair::from(Keyring::from_public(Public::from_raw(extrinsic.signed.0.clone())).unwrap())
-				.sign(&extrinsic.encode()).into());
-			let extrinsic = Extrinsic {
-				signed: extrinsic.signed.into(),
-				index: extrinsic.index,
-				function: extrinsic.function,
-			};
-			UncheckedExtrinsic::new(extrinsic, signature)
-		}).collect::<Vec<_>>();
-
-		let extrinsics_root = ordered_trie_root(extrinsics.iter().map(Slicable::encode)).0.into();
+		let extrinsics = extrinsics
```

Comment:
our house style of keeping _some chained_ methods in the first line and wrap for a bigger block isn't really something that I was able to configure in rustfmt. While I do prefer our house style here, I'd not complain about the rustfmt style either.

Reply:
This is one of the rules where I felt rustfmt was just too inflexible when I tried it the last time. Apparently nothing has changed there. :/